string model
string method
string exposure
string save_path
---
bool success
